api/coreservice.go

```diff
@@ -1213,6 +1217,58 @@ func (core *coreService) getBlockByHeight(height uint64) (*apitypes.BlockWithRec
 	}, nil
 }
 
+func (core *coreService) BlobSidecarsByHeight(height uint64) ([]*apitypes.BlobSidecarResult, error) {
+	header, err := core.bc.BlockHeaderByHeight(height)
+	if err != nil {
+		return nil, err
+	}
+	res := make([]*apitypes.BlobSidecarResult, 0)
+	blobs, txHashes, err := core.getBlobSidecars(height)
+	if err != nil {
+		return nil, err
+	}
+	txIndices := make([]uint64, 0)
+	for _, txHash := range txHashes {
+		_, _, index, err := core.ActionByActionHash(txHash)
+		if err != nil {
+			return nil, err
+		}
+		txIndices = append(txIndices, uint64(index))
+	}
+	blkHash := header.HashBlock()
+	for i, blob := range blobs {
+		res = append(res, &apitypes.BlobSidecarResult{
+			BlobSidecar: blob,
+			BlockNumber: height,
+			BlockHash:   common.BytesToHash(blkHash[:]),
```

Comment:
`BlockNumber, BlockHash` is common for all tx, only need 1 copy? i mean like below?
```
apitypes.BlobSidecarResult {
    BlockNumber
    BlockHash
    [] struct {
        blobSidecar
        txIndex
        txHash
    }
}
```

Reply:
I think we can just keep same as [bsc](https://github.com/bnb-chain/bsc/blob/master/core/types/blob_sidecar.go#L25). IMO, this design is to use same structure for getSidercarByBlock and getSidercarByTx